components/SearchBar/index.js

```diff
@@ -48,16 +50,17 @@ function SearchBar({ search, onSearch }) {
         <Styled.FilterButton
           onClick={() => setIsFilter((isFilter) => !isFilter)}
         >
-          Filter
+          <FaSliders />
         </Styled.FilterButton>
       </Styled.SearchAndFilter>
       {isFilter && (
         <Styled.FilterBox>
           <Styled.FilterContainer>
             <Styled.FilterItem>
-              <label>Category</label>
+              <label for="category">Category</label>
```

Comment:
[We need to use `htmlFor` in React](https://legacy.reactjs.org/docs/dom-elements.html#htmlfor).